src/huggingface_hub/hf_api.py

```diff
@@ -939,8 +952,10 @@ def model_info(
         d = r.json()
         return ModelInfo(**d)
 
+    @_deprecate_positional_args
     def list_repo_files(
         self,
+        *,
         repo_id: str,
```

Comment:
```suggestion
    @_deprecate_positional_args
    def list_repo_files(
        self,
        repo_id: str,
        *,
```

Reply:
There are a few others below I can't do with suggestions, so will stop there for suggestions on this kind of reordering.

Reply:
There are a few others below I can't do with suggestions, so will stop there for suggestions on this kind of reordering.